libs/designsystem/src/lib/components/list/list-experimental/list-experimental.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'kirby-list-experimental',
+  templateUrl: './list-experimental.component.html',
```

Comment:
I like that we're not including any css files as of now. Truly separating structure from styling.

Reply:
Wohooo 💯